frontend/src/Frontend/Wallet.hs

```diff
@@ -125,6 +128,11 @@ instance FromJSON key => FromJSON (SomeAccount key) where
   parseJSON Null = pure SomeAccount_Deleted
   parseJSON x = SomeAccount_Account <$> parseJSON x
 
+activeAccountOnNetwork :: NetworkName -> SomeAccount key -> Maybe (Account key)
+activeAccountOnNetwork net = someAccount
```

Comment:
Deployment settings should use this once !271 is merged.

Reply:
Does this need taking care of in this PR?

Reply:
Not really. It shouldn't hold this up. It was just a nice to do given it abstracts it.